relay-spans/src/span.rs

```diff
@@ -103,6 +107,23 @@ pub fn otel_to_sentry_span(otel_span: OtelSpan) -> EventSpan {
             };
             if key == "sentry.op" {
                 op = otel_value_to_string(value);
+            } else if key.starts_with("db") {
+                op = op.or(Some("db".to_string()));
+                if key == "db.statement" {
+                    if let Some(statement) = otel_value_to_string(value) {
+                        description = statement;
+                    }
+                }
+            } else if key == "http.method" || key == "request.method" {
+                let http_op = match kind {
+                    2 => "http.server",
+                    3 => "http.client",
+                    _ => "http",
+                };
+                op = op.or(Some(http_op.to_string()));
+                http_method = otel_value_to_string(value);
+            } else if key == "http.route" || key == "url.path" {
+                http_route = otel_value_to_string(value);
```

Comment:
If you only use the `http_route` to build the description via `format`, you don't have to allocate here.

`otel_value_to_string` can return `Cow<'a, str>` and whenever you end up needing an actual `String` call `into_owned()` on it.

Reply:
i'm not sure what that implies in terms of what all i need to change, feel free to commit to the PR

Reply:
No worries, let's ignore it for now.